src/core/api/repository.go

```diff
@@ -649,6 +649,7 @@ func assembleTagsInParallel(client *registry.Repository, projectID int64, reposi
 		}
 	}
 
+	limit := make(chan bool, 100)
```

Comment:
100 is determined by the DB connection max number?

Reply:
just an experience value